Lib/venv/scripts/nt/activate.bat

```diff
@@ -21,8 +21,8 @@ set PROMPT=__VENV_PROMPT__%PROMPT%
 if defined PYTHONHOME set _OLD_VIRTUAL_PYTHONHOME=%PYTHONHOME%
 set PYTHONHOME=
 
-if defined _OLD_VIRTUAL_PATH set PATH=%_OLD_VIRTUAL_PATH%
-else set _OLD_VIRTUAL_PATH=%PATH%
+if defined _OLD_VIRTUAL_PATH set "PATH=%_OLD_VIRTUAL_PATH%"
+if not defined _OLD_VIRTUAL_PATH set "_OLD_VIRTUAL_PATH=%PATH%"
```

Comment:
Need to lose the quotes as well - they can cause problems if there are mismatched quotes in the variables (and they're not necessary when the statement extends to the end of the line, which is why the original change was necessary).